src/components/MembersPanel.tsx

```diff
@@ -368,9 +368,31 @@ const MembersPanel = ({ members, setMembers }: MembersPanelProps) => {
       );
    }
 
+   function totalMembers(): number {
+      return (
+         members.length
+      );
+   }
+
+   function activeMembers(): number {
+      let activeTotal = 0
+
+      for (let member of members) {
+         if (member.active) {
```

Comment:
Up to you as a developer, but I prefer single-line if statements if the body is one line.

=> if (member.active) activeTotal...